### PR TITLE
Master

### PR DESCRIPTION
**Pull request by GeoffryAbiFarah**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by GeoffryAbiFarah** [comment]
Is there something else I am supposed to do?

File: certified-connectors/Mailjet/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "IsExcludedFromCampaigns": {
+                  "type": "string",
+                  "description": "IsExcludedFromCampaigns"
```
Comment (sriyen-msft) [inline certified-connectors/Mailjet/apiDefinition.swagger.json:63]: @GeoffryAbiFarah All the description values of properties should be free from grammatical errors.

File: certified-connectors/Mailjet/readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Authentication:
```
Comment (sriyen-msft) [inline certified-connectors/Mailjet/readme.md:1]: @GeoffryAbiFarah Can you please refer a sample readme.md [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md) and improve your connector's readme.md

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Mailjet/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Javista",
```
Comment (sriyen-msft) [inline certified-connectors/Mailjet/apiProperties.json:33]: @GeoffryAbiFarah Can you please share a document or any proof that `Javista` is authorized to create connector for `MailJet` ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@GeoffryAbiFarah I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@GeoffryAbiFarah I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@GeoffryAbiFarah I am closing this PR as there is no response from you since 27 days and you can reopen this same PR when you are ready for certification. Thanks

**Pull request closed without merging**